core/src/HIP/Kokkos_HIP_Locks.cpp

```diff
@@ -56,8 +56,7 @@ namespace Kokkos {
 
 #ifdef KOKKOS_ENABLE_HIP_RELOCATABLE_DEVICE_CODE
 namespace Impl {
-__device__ __constant__ HIPLockArrays g_device_hip_lock_arrays = {nullptr,
-                                                                  nullptr, 0};
+__device__ __constant__ HIPLockArrays g_device_hip_lock_arrays = {nullptr, 0};
```

Comment:
For reference
https://github.com/kokkos/kokkos/blob/cb636c3344a06918c4aeaa7c53aec251f8c35c04/core/src/HIP/Kokkos_HIP_Locks.hpp#L61-L64